### DISCUSSION TITLE
Is there a way to force a telnet-client inside electerm to send CR instead of CRLF?

### DISCUSSION THREAD

**Discussion by furriest** [electerm/electerm#3495] [Q&A]
In my install, the electerm-web sends CRLF in telnet sessions, and the connecting server translates it as double enter.
If I use telnet client on Windows, I can use an "unset crlf" command, which solves the issue. 
Is there a configuration parameter I can change to send only CR in my telnet connections from electerm?

**Comment by zxdong262** [comment]
https://github.com/electerm/electerm/issues/3496 tracked here, would investigate how to handle it, thank you for the feedback!
